rawnode_test.go

```diff
@@ -224,13 +219,11 @@ func TestRawNodeProposeAndConfChange(t *testing.T) {
 		},
 	}
 
-	for _, tc := range testCases {
+	for i, tc := range testCases {
 		t.Run("", func(t *testing.T) {
 			s := newTestMemoryStorage(withPeers(1))
 			rawNode, err := NewRawNode(newTestConfig(1, 10, 1, s))
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err, "#%d", i)
```

Comment:
This `#%d` printing is unnecessary. The purpose of this printing is to be able to identify which subtest/iteration failed when debugging test failures. However, in this case, every subtest is wrapped in `t.Run()` which makes it a separate test with a unique name. In this case, the test name will be an incremental index. If a subtest fails, the error message will contain this subtest name.